fares-data-build-tool/fdbt-site
# Feature/tfn 699 cookie preferences

# Description

-   This PR adds a new page that allows a user to set their cookie preferences. It also adds a banner which prompts the user to set their cookie preferences if they haven't already done so. Furthermore, this PR disables Google Analytics by default, unless the user allows tracking when they set their cookie preferences.

# Testing instructions

-   Pull down this branch and run the site locally. Check that you can navigate to the new cookie preferences page via the footer or banner link and that the new page behaves as expected. Also, validate that the cookie banner displays only when cookie preferences haven't been set.

# Type of change


# Checklist:
